packages/mobile/__mocks__/@celo/contractkit/index.ts

```diff
@@ -0,0 +1,45 @@
+import BigNumber from 'bignumber.js'
+
+export const newKitFromWeb3 = () => ({
```

Comment:
I would suggest keeping a contract kit mock in the contract kit package so we can keep it up to date and then have consumers like mobile import it here